addons/sale_management/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, api, _
```

Comment:
```suggestion
from odoo import api, models, _
```

Alphabetical order of imports (https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#imports)